tests/features/api.feature

```diff
@@ -0,0 +1,283 @@
+        | Reference                 | My Reference                                     |
+        | Project period start date | 2020-01-01                                       |
+        | Project period end date   | 2020-01-02                                       |
+        | Overall budget            | 100000                                           |
+        | EU contribution           | 50000                                            |
+        | Website                   | uri: http://ec.europa.eu/info - title: Info site |
+        | Body text                 | My Body text                                     |
+        | Call for proposals        | uri: http://ec.europa.eu/info - title: Info site |
+        | Results                   | My Results                                       |
+        | Teaser                    | My teaser                                        |
+        | Coordinators              | A Stakeholder                                    |
+        | Departments               | Asian Development Bank                           |
+        | Featured media            | Image placeholder                                |
+        | Funding programme         | Anti Fraud Information System (AFIS)             |
+        | Participants              | A Participant                                    |
+        | Project contact           | A general contact                                |
+        | Documents                 | Document placeholder                             |
+        | Result files              | Document placeholder                             |
+
+      When I visit node "My project" edit page
+      Then I should see the text "My project"
+      And I should see the text "My Summary"
+      And the "Reference" field should contain "My Reference"
+      And the "oe_project_dates[0][value][day]" field should contain "1"
+      And the "oe_project_dates[0][value][month]" field should contain "1"
+      And the "oe_project_dates[0][value][year]" field should contain "2020"
+      And the "oe_project_dates[0][end_value][day]" field should contain "2"
+      And the "oe_project_dates[0][end_value][month]" field should contain "1"
+      And the "oe_project_dates[0][end_value][year]" field should contain "2020"
+      And the "Overall budget" field should contain "100000"
+      And the "EU contribution" field should contain "50000"
+      And the "oe_project_website[0][uri]" field should contain "http://ec.europa.eu/info"
+      And the "oe_project_website[0][title]" field should contain "Info site"
+      And the "Body text" field should contain "My Body text"
+      And the "oe_project_calls[0][uri]" field should contain "http://ec.europa.eu/info"
+      And the "oe_project_calls[0][title]" field should contain "Info site"
+      And the "Results" field should contain "My Results"
+      And the "Teaser" field should contain "My teaser"
+      And I should see the text "A Stakeholder"
```

Comment:
we should check for all avail values in the inline entity form for stackeholders, participants and general contact